builtInFunctions/changeOwnerAddress.go

```diff
@@ -81,6 +93,21 @@ func (c *changeOwnerAddress) ProcessBuiltinFunction(
 	return vmOutput, nil
 }
 
+func (c *changeOwnerAddress) addOutputTransferToVmOutputForCallThroughSC(acntDst vmcommon.UserAccountHandler, vmInput *vmcommon.ContractCallInput, vmOutput *vmcommon.VMOutput) {
+	isCrossShardCallThroughASmartContract := check.IfNil(acntDst) && vmcommon.IsSmartContractAddress(vmInput.CallerAddr)
```

Comment:
It is easier to read if we have early exists.
if flagNotEnabled return
if NotCrossShardThroughSC return

Reply:
done